src/Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorApp.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Column, GridColumns, Spacer, Text } from "@artsy/palette"
+import { ERROR_MESSAGES } from "Components/ErrorPage"
+import { ArtworkErrorApp_artworkError$data } from "__generated__/ArtworkErrorApp_artworkError.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+import { OtherWorksQueryRenderer } from "Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorAppOtherWorks"
+import { RelatedWorksQueryRenderer } from "Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorAppRelatedWorks"
+
+interface ArtworkErrorAppProps {
+  artworkError: ArtworkErrorApp_artworkError$data
+}
+const ArtworkErrorApp = (props: ArtworkErrorAppProps) => {
+  const {
+    artworkError: { requestError, artwork },
+  } = props
+  const statusCode = requestError?.statusCode ?? 500
+  const headline = ERROR_MESSAGES[statusCode] ?? "Internal Error"
+
+  const artworkSlug = artwork?.slug
+
+  return (
+    <Box>
+      <Spacer y={6} />
```

Comment:
https://github.com/artsy/force/blob/main/docs/best_practices.md#avoid-external-margins

Reply:
I should add a caveat for if its literally the first thing on the page :)